infrastructure/game-client-startup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Script to install applications for the Game Client VM 
+
+export DEBIAN_FRONTEND=noninteractive
+
+wget -O "/usr/share/keyrings/xpra.asc" https://xpra.org/xpra.asc
+cd /etc/apt/sources.list.d 
+wget https://raw.githubusercontent.com/Xpra-org/xpra/master/packaging/repos/bookworm/xpra.sources
+sudo apt update -y
+cd -
+
+curl -OL https://github.com/LizardByte/Sunshine/releases/download/v0.21.0/sunshine-debian-bookworm-amd64.deb
+sudo apt install -f ./sunshine-debian-bookworm-amd64.deb -y
+sudo apt install xfce4 xfce4-goodies tightvncserver dbus-x11 xpra -y
```

Comment:
We should also add the nvidia proprietary drivers as well, since we'll need them to run the game.

https://wiki.debian.org/NvidiaGraphicsDrivers

Reply:
May also want to remove in and run `glxgears -info` just to see that the display driver is working correctly.